### PR TITLE
Add a simple tutorial about server-side rendering

### PR DESCRIPTION
**Pull request by zaaack**
Sorry that I forgot to handle null elements in the last PR, 😅 this PR will fix it.

Also, I wrote a simple tutorial about server-side rendering, not sure it's clear enough.

### CODE REVIEW

File: docs/server-side-rendering.md [hunk 1]
```diff
@@ -0,0 +1,210 @@
...
+#endif
+-|> Program.withReact "elmish-app"
++|> Program.withReactHydrate "elmish-app"
+#if DEBUG
+|> Program.withDebugger
+#endif
+|> Program.run
+```
+
+Now enjoi! If you find bugs or just need some help, please create an issue and let we know.
```
Comment (MangelMaxime) [inline docs/server-side-rendering.md]: I think it's:

Now enjoy

Comment (zaaack) [reply docs/server-side-rendering.md]: Aha, thanks! 😅

**Comment by forki** [comment]
oups I did something stupid. I fixed couple of typos but pushed it to main fork

File: docs/server-side-rendering.md (hunk 1 above)
Comment (forki) [reply docs/server-side-rendering.md]: already fixed

**Pull request merged**